src/SOS/Strike/sos.cpp

```diff
@@ -803,9 +803,10 @@ namespace sos
         // there are no SyncBlocks in the process.
         DacpSyncBlockData syncBlockData;
         if (SUCCEEDED(syncBlockData.Request(g_sos, 1)))
+        {
             mTotal = syncBlockData.SyncBlockCount;
-
-        mSyncBlk = mCurr;
+            mSyncBlk = mCurr;
```

Comment:
So the bug was ours all along?

Reply:
Yes. It (setting mSyncBlk) threw an exception when there are no sync blocks.